clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -2774,6 +2774,9 @@ void handleResponse(AbstractResponse abstractResponse) {
                             context.node().get(),
                             authorizedOperations);
                     context.future().complete(consumerGroupDescription);
+                } else {
+                    context.future().completeExceptionally(new IllegalArgumentException(
+                        "GroupId " + context.groupId() + " is not a consumer group."));
```

Comment:
For completeness, could we add the protocolType to this message?

Reply:
Yep. Thank for the suggestion.